### PR TITLE
Stephanie50

### PR DESCRIPTION
**Pull request by DevStephanie**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigquery-dataframes/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: samples/snippets/create_multiple_timeseries_forecasting_model.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
+def test_multiple_timeseries_forecasting_model(random_model_id):
+    # [START bigquery_dataframes_bqml_create_data__set]
```
Comment (shobsi) [inline samples/snippets/create_multiple_timeseries_forecasting_model.py:2]: typo (double underscore)?

File: samples/snippets/create_multiple_timeseries_forecasting_model.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+    # Start by selecting the data you'll use for training. `read_gbq_table` accepts
+    # either a SQL query or a table ID. Since this example selects from multiple
+    # tables via a wildcard, use SQL to define this data. Watch issue
+    # https://github.com/googleapis/python-bigquery-dataframes/issues/169
+    # for updates to `read_gbq_table` to support wildcard tables.
+
+    df = bpd.read_gbq_table("bigquery-public-data.new_york.citibike_trips", filters=[])
+
+    # [END bigquery_dataframes_bqml_create_data__set(1)]
```
Comment (shobsi) [inline samples/snippets/create_multiple_timeseries_forecasting_model.py:16]: I think it has to be the exact same tag as START

File: samples/snippets/create_multiple_timeseries_forecasting_model.py [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+    # LINE GRAPH GOES HERE
+    # EXPLAIN WHY WER
+    num_trips.plot.line(
+        rot=45,
+    )
+
+    # [END bigquery_dataframes_bqml_visualize_time_series_to_forecast]
+
+    # [START bigquery_dataframes_bqml_visualize_time_series_to_forecast]
```
Comment (shobsi) [inline samples/snippets/create_multiple_timeseries_forecasting_model.py:67]: Should use a unique tag

File: samples/snippets/create_multiple_timeseries_forecasting_model.py [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+        your_model_id,  # For example: "bqml_tutorial.sample_model",
+        replace=True,
+    )
+    # ARIMAPlus(auto_arima_max_order=5, data_frequency='AUTO_FREQUENCY',
+    # max_time_series_length=3, min_time_series_length=20,
+    # time_series_length_fraction=1.0, trend_smoothing_window_size=-1)
+
+    # [END bigquery_dataframes_bqml_visualize_time_series_to_forecast]
+
+    # [START bigquery_dataframes_bqml_visualize_time_series_to_forecast]
```
Comment (shobsi) [inline samples/snippets/create_multiple_timeseries_forecasting_model.py:98]: should use a unique tag

Comment (DevStephanie) [reply samples/snippets/create_multiple_timeseries_forecasting_model.py:98]: yes, agree unique tags per section.

File: samples/snippets/create_multiple_timeseries_forecasting_model.py (hunk 3 above)
Comment (DevStephanie) [reply samples/snippets/create_multiple_timeseries_forecasting_model.py:67]: yes, agree unique tags per section.

**Pull request closed without merging**